sys/net/gnrc/network_layer/ipv6/ext/frag/gnrc_ipv6_ext_frag.c

```diff
@@ -40,6 +40,7 @@ static gnrc_ipv6_ext_frag_limits_t _limits_pool[CONFIG_GNRC_IPV6_EXT_FRAG_LIMITS
 static clist_node_t _free_limits;
 static xtimer_t _gc_xtimer;
 static msg_t _gc_msg = { .type = GNRC_IPV6_EXT_FRAG_RBUF_GC };
+static gnrc_ipv6_ext_frag_stats_t _stats;
```

Comment:
```suggestion
#if IS_USED(MODULE_GNRC_IPV6_EXT_FRAG_STATS)
static gnrc_ipv6_ext_frag_stats_t _stats;
#endif
```

Reply:
Mhhhh my comment on this somehow disappeared. So here it is again.

This wouldn't work, as I use `if (IS_USED(…))` whenever I access this variable below, so guarding this will lead to compile-time errors. The optimizer will throw this out, as it is static and only used with `IS_USED(…)`.

Reply:
I just noticed that too - I thought Murdock would complain about the unused variable, but the `IS_USED(…)` prevented this.